src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
```

Comment:
The conditional days should be stored in a way to facilitate changes - also remember the naming guidelines (check 1) when you do this
☝️
[checklist 2](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)